docs/site/todo-list-tutorial-sqldb.md

```diff
@@ -0,0 +1,134 @@
+---
+lang: en
+title: 'Running on relational databases'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-sqldb.html
+summary:
+  LoopBack 4 TodoList Application Tutorial - Running on Relational Databases
+---
+
+If you are running this example using a relational database, there are extra
+steps in order to set up constraints in the database. We're using PostgreSQL for
+illustration, but it would work for other relational databases in a similar way.
+
+### Create a new datasource
+
+We're going to reuse the DataSource `db` to connect to the PostgreSQL database
```

Comment:
Nitpick: I would personally use "rework" or "update" instead of "reuse".

```suggestion
We're going to update the DataSource `db` to connect to the PostgreSQL database
```